packages/@okta/vuepress-site/docs/guides/quickstart/try-api/java/try-api.md

```diff
@@ -0,0 +1,5 @@
+To create a user by means of the Java SDK:
+
+1. 
+
```

Comment:
@bdemers, could you help me fill in this section with information on creating a new user via the Java SDK?